Makefile

```diff
@@ -0,0 +1,18 @@
+nonprod:
+	cd auacm/angular/ && ng build --deploy-url static/
+	cp auacm/angular/dist/index.html auacm/app/templates/index.html
```

Comment:
I know that this is a small Makefile, but for maintainabilty, it might be better to refactor the duplicated rules into one rule that gets called.